etc/behat/services/contexts.xml

```diff
@@ -24,6 +24,8 @@
         <parameter key="sylius.behat.context.setup.shipping.class">Sylius\Behat\Context\Setup\ShippingContext</parameter>
         <parameter key="sylius.behat.context.setup.zone.class">Sylius\Behat\Context\Setup\ZoneContext</parameter>
         <parameter key="sylius.behat.context.setup.tax.class">Sylius\Behat\Context\Setup\TaxContext</parameter>
+        <parameter key="sylius.behat.context.setup.lexical.class">Sylius\Behat\Context\Setup\LexicalContext</parameter>
```

Comment:
IMHO new namespace `sylius.behat.context.transform.*` would be the most suitable for contexts that only transforms arguments